src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -87,7 +88,7 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
             </TextContent>
             <ClusterHostsTable cluster={cluster} skipDisabled />
           </GridItem>
-          <ClusterProperties cluster={cluster} />
+          <ClusterProperties cluster={cluster} isDualStack={isDualStack(cluster)} />
```

Comment:
If ClusterProperties receives the whole cluster obj, why does it need an additional `isDualStack` prop? :)